deploy/scripts/src/fraud/test.ts

```diff
@@ -0,0 +1,93 @@
+import { type Options } from 'k6/options'
+import {
+  selectProfile,
+  type ProfileList,
+  describeProfile,
+  createScenario,
+  LoadProfile
+} from '../common/utils/config/load-profiles'
+import { getAccessToken } from '../cri-kiwi/utils/authorization'
```

Comment:
If you need this function it should probably be moved to a common folder, as it's no longer just applicable to kiwi